addons/account/static/src/components/section_and_note_fields_backend/section_and_note_backend.scss

```diff
@@ -2,40 +2,65 @@
 // The goal of this file is to contain CSS hacks related to allowing
 // section and note on sale order and invoice.
 
-table.o_section_and_note_list_view tr.o_data_row {
-    &.o_is_line_note {
-        font-style: italic;
-    }
-    &.o_is_line_section {
-        font-weight: bold;
-        background-color: #DDDDDD;
-        --table-bg: var(--300);
-    }
-    &.o_is_line_subsection {
-        --table-bg: var(--200);
-    }
-    &.o_is_line_section, &.o_is_line_subsection {
-        border-top: 1px solid #BBB;
-        border-bottom: 1px solid #BBB;
-        .o_list_section_options {
-            width: 1px;  // to prevent the column to expand
-            button {
-                padding: 0px;
-                background: none;
-                border-style: none;
-                display: table-cell;
-                cursor: pointer;
+table.o_section_and_note_list_view {
+    --o-section-border-color: #{$gray-500};
+    --o-section-hover-effect-transition: 0.1s ease-in-out;
+
+    tr {
+        &.o_is_line_note {
+            font-style: italic;
+        }
+
+        &.o_is_line_section, &.o_is_line_subsection {
+            --table-striped-bg: var(--table-bg-type);
+
+            .o_list_section_options {
+                width: 1px;  // to prevent the column to expand
+            }
+
+            td[name="price_subtotal"] {
+                color: $gray-700;
             }
         }
-    }
-    &.o_is_line_note,
-    &.o_is_line_section,
-    &.o_is_line_subsection {
-        td {
-            // There is an undeterministic CSS behaviour in Chrome related to
-            // the combination of the row's and its children's borders.
-            border: none !important;
+
+        &.o_is_line_section {
+            --table-hover-bg: #{rgba($body-emphasis-color, .08)};
+            --table-bg-type: #{rgba($body-emphasis-color, .06)};
+
+            .o_handle_cell .o_row_handle {
+                transition: opacity var(--o-section-hover-effect-transition);
+            }
+
+            &:where(:not(.o_selected_row)) {
+                div:has(> div.o_field_product_label_section_and_note_cell) {
+                    transition: transform var(--o-section-hover-effect-transition);
+                    position: absolute;
+                }
+
+                &:not(:hover, .o_dragged) {
+                    div:has(> div.o_field_product_label_section_and_note_cell) {
+                        transform: translateX(calc((var(--body-font-size) + (#{$table-cell-padding-x-sm} * 2)) * -1));
+                    }
+
+                    .o_handle_cell .o_row_handle {
+                        opacity: 0;
+                    }
+                }
+            }
         }
+
+        &.o_is_line_subsection {
+            --table-hover-bg: #{rgba($body-emphasis-color, .05)};
+            --table-bg-type: #{rgba($body-emphasis-color, .03)};
+        }
+
+        &:where(:not(.o_dragged):has(+ .o_is_line_section:not(.o_dragged, .d-table-row))) {
```

Comment:
wdyt about "saving" these complex selector in an human-readible scss variable? eg

```scss

$_isFirstRow-selector: ':not(.o_dragged):has(+ .o_is_line_section:not(.o_dragged, .d-table-row))';

&:where(#{$_isFirstRow}) {
}
```

Reply:
They are unique selectors, I don't think we benefit from creating a variable if it's not to be used at multiple places, I do think it deserves a comment

Reply:
I'm ok with the comment